scripts/copy_from_pqclean/copy_from_pqclean.py

```diff
@@ -23,12 +31,11 @@ def file_put_contents(filename, s, encoding=None):
 def generator(destination_filename, template_filename, family, scheme_desired):
     template = file_get_contents(os.path.join('scripts', 'copy_from_pqclean', template_filename))
     f = copy.deepcopy(family)
+    #if (destination_filename == "src/kem/kyber/kem_kyber_512.c"): 
```

Comment:
Is this line still needed?

Reply:
Absolutely no. Removed and merged with upstream.